### ISSUE TITLE
Value Error: numpy.ndarray size changed, may indicate binary incompatibility. Expedected 96 from C header, got 88 from PyObject

### ISSUE THREAD

**Issue by XiangPan-KimLab** [theia-dev/Qiber3D#1]
Hello, I recently found the brilliant tool you developed, named Qiber 3D. However, as I was going through your example code, I faced this error when generating this code: 

import logging
from Qiber3D import Network, config
from Qiber3D.helper import Example

The error is attached as an image:
![unnamed](https://github.com/theia-dev/Qiber3D/assets/150743489/992e2e3c-9021-4037-aa8a-b4781b477705)

**Comment by JannikIrmai** [comment]
This seems to be a problem with the numpy version. For me, upgrading numpy resolved the problem:
```
 pip install --upgrade numpy
```

**Comment by JannikIrmai** [comment]
I had the same problem and it seems to be caused by incompatible package versions. I could resolve the problem by using python 3.7 (instead of 3.8).

**Comment by nathan-diodan** [comment]
Yes, the current release is problematic on the latest python version. Please try the `dev` version with updated requirements and let me know if it worked in your environment.

```
pip install -U git+https://github.com/theia-dev/Qiber3D.git@dev#egg=Qiber3D
```

Best Hagen
